src/app.ts

```diff
@@ -185,6 +185,7 @@ export class App {
   validator = new Validator();
 
   _contextClass: any = class extends ContextWrapper {};
+  _nestedFunctions: Record<string, Record<string, MojoAction>> = {};
```

Comment:
`_nestedHelpers`?

Reply:
fixed using _nestedHelpersCache

Reply:
No, i mean `_nestedHelpers` would be a more appropriate name here.

Reply:
Nooooo, i meant here! `_nestedFunctions` should be `_nestedHelpers`.

Reply:
should be ok now, _nestedHelpers is the prop in the app object, and _nestedHelpersCache the one in the ctx